### ISSUE TITLE
Too specific dependencies for open Suse leap

### ISSUE THREAD

**Issue by atdotde** [subsurface/subsurface#924]
I had a user report on Facebook that our rpm packages for openSUSE Leap 43.2 (he probably meant 42.3) require qt in version 5.9.2 but he has 5.9.3 and says we should only require 5.9

I have no idea what we actually do need and what we should require, just reporting. 

### Describe the issue:

- [x] Bug
- [ ] Change request
- [ ] New feature request
- [ ] Discussion request

**Comment by dirkhh** [comment]
I don't think we are doing that directly. That's an OBS artifact. Similar to the Fedora issue in the user forum
